src/User.hpp

```diff
@@ -6,6 +6,8 @@
 class User
 {
 public:
+    User() = default;
```

Comment:
we don't want to have this construtor. Please remove it.

Reply:
Done